int32 crowdNum
float32 centrex
float32 centrey
float32 manWidth